.github/workflows/build.yml

```diff
@@ -339,3 +339,31 @@ jobs:
           RCLONE_CONFIG_PASS: ${{ secrets.RCLONE_CONFIG_PASS }}
         # Upload artifacts if not a PR && not a fork
         if: github.head_ref == '' && github.repository == 'rclone/rclone'
+
+  ios:
+    if: ${{ github.repository == 'rclone/rclone' || github.event.inputs.manual }}
+    timeout-minutes: 30
+    name: "ios-all"
```

Comment:
This should maybe be called ios-simulator?

Reply:
Good idea!